app/javascript/components/ingredients/ingredients-container.vue

```diff
@@ -227,8 +230,11 @@ export default {
     async editIngredient() {
       this.showingEdit = !this.showingEdit;
       try {
-        const res = await editIngredient(this.ingredientToEdit.id, this.$refs.editIngredientInfo.form);
-        this.updateIngredient(res);
+        var ingredientsInfo = this.$refs.editIngredientInfo.form;
+        ingredientsInfo.ingredient_measures_attributes = ingredientsInfo
```

Comment:
🚫 **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Identifier 'ingredient_measures_attributes' is not in camel case  camelcase